content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -85,31 +85,30 @@ In order to build Atom from source, you need to have a number of other requireme
 * macOS 10.9 or later
 * Node.js 6.x or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
 * npm 3.10.x or later (run `npm install -g npm`)
-* Python v2.7.x
+* Python v2.7.x or 3.5+
 * Command Line Tools for [Xcode](https://developer.apple.com/xcode/downloads/) (run `xcode-select --install` to install)
 
 {{/mac}}
 
 {{#windows}}
 
 * Node.js 6.9.4 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
-* Python v2.7.x
+* Python v2.7.x or 3.5+
   * The `python.exe` must be available at `%SystemDrive%\Python27\python.exe`. If it is installed elsewhere create a symbolic link to the directory containing the `python.exe` using: `mklink /d %SystemDrive%\Python27 D:\elsewhere\Python27`
```

Comment:
This part is sadly still true on Windows, until https://github.com/atom/atom/pull/20856 is merged.

The bootstrap script errors out if Python 2.7 isn't found at `[Drive_root]:\Python27\python.exe`. This is hard-coded in Atom's `script/lib/verify-machine-requirements.js` sub-script. My PR that I just linked to above fixes that.

Reply:
atom/atom#20856 was merged! 🎉